node/storagenode.go

```diff
@@ -45,48 +46,48 @@ func (node *Node) QueryDNS(website string) {
 	node.CacheTime += 1
 
 	if strings.HasPrefix(website, "www.") {
-		system.Println("> Removing Prefix")
+		log.Info().Msg("Removing Prefix")
 		website = website[4:]
 	}
 	hashedWebsite := utility.GenerateHash(website)
 	ip_addr, ok := node.CachedQuery[hashedWebsite]
 	if ok {
-		system.Println("> Retrieving from LRUCache")
+		log.Info().Msg("Retrieving from LRUCache")
 		for _, ip_c := range ip_addr.value {
-			system.Printf("> %s. IN A %s\n", website, ip_c)
+			log.Info().Msgf("> %s. IN A %s", website, ip_c)
 		}
 	} else {
 		ip_addr, ok := node.HashIPStorage[node.Nodeid][hashedWebsite]
-		system.Printf("> The Website %s has been hashed to %d\n", website, hashedWebsite)
+		log.Info().Msgf("> The Website %s has been hashed to %d", website, hashedWebsite)
 		if ok {
-			system.Println("> Retrieving from Local Storage")
+			log.Info().Msg("Retrieving from Local Storage")
 			for _, ip_c := range ip_addr {
-				system.Printf("> %s. IN A %s\n", website, ip_c)
+				log.Info().Msgf("> %s. IN A %s", website, ip_c)
 			}
 		} else {
 			succPointer, hopCount := node.FindSuccessor(hashedWebsite, 0)
-			system.Println("> Number of Hops:", hopCount)
+			log.Info().Msgf("> Number of Hops: %d", hopCount)
 			// log hopcount into the log file using the library
-			system.Printf(">  The Website would be stored at it's succesor %d : %s\n", succPointer.Nodeid, succPointer.IP)
+			log.Info().Msgf("> The Website would be stored at it's succesor Nodeid: %d IP: %s", succPointer.Nodeid, succPointer.IP)
 			msg := message.RequestMessage{Type: GET, TargetId: hashedWebsite}
 			reply := node.CallRPC(msg, succPointer.IP)
 			if reply.QueryResponse != nil {
-				system.Println("> Retrieving from Chord Network")
+				log.Info().Msg("Retrieving from Chord Network")
 				for _, ip_c := range reply.QueryResponse {
-					system.Printf("> %s. IN A %s\n", website, ip_c)
+					log.Info().Msgf("> %s. IN A %s", website, ip_c)
 				}
 			} else {
 				ips, err := net.LookupIP(website)
 				if err != nil {
-					system.Printf("> Could not get IPs: %v\n", err)
+					log.Error().Err(err).Msg("Could not get IPs")
 					return
 				}
 				ip_addresses := []string{}
-				system.Println("IP ADDRESSES", ip_addresses)
+				log.Info().Msgf("IP ADDRESSES %v", ip_addresses)
 
 				for _, ip := range ips {
 					ip_addresses = append(ip_addresses, ip.String())
-					system.Printf("> %s. IN A %s\n", website, ip.String())
+					log.Info().Msgf("> %s. IN A %s", website, ip.String())
 				}
```

Comment:
I think most of these from `QueryDNS` can be moved to DEBUG, if DEBUG is less severe than INFO. We don't need to see these messages unless we're debugging right?
wdyt
@anirhc @64hann @s4nat @rmurarishetti